spec/game_spec.rb

```diff
@@ -65,4 +65,57 @@ def print_message(string)
       expect(console.print_message(console.printed_string)).to eq(expected_output)
     end
   end
+
+  describe 'game_over?' do
+    it 'returns true if there is a winning board' do
+      board = Board.new
+      console = FakeConsole.new
```

Comment:
I wouldn't expect to need to create a fake console in order to test the `game_over` method... what does this tell you about the responsibility of `Game`/the way your classes are set up?

Reply:
Would you suggest that when my Game class instantiates that it can have a default argument?

Reply:
No - that wouldn't be helpful here because you'd still have to mock out the console. I'm suggesting that you might need to make another class for something because it doesn't make sense that the `game_over` method implicitly depends on having a console.